db/block_internal.h

```diff
@@ -629,6 +629,21 @@ enum { FSTBLK_RSPKL_LEN = 8 };
 BB_COMPILE_TIME_ASSERT(fstblk_rspkl_size,
                        sizeof(struct fstblk_rspkl) == FSTBLK_RSPKL_LEN);
 
+/* This buffer must always be able to hold a fstblk header and the max number
+   of block err's. It will also have to hold one of the following:
+     fstblk pre rspkl + rspkl,
+     fstblk rsperr, or
+     fstblk rspok.
+   Since I don't want to bother figuring out which of those lengths is the
+   longest, just add them all together.
+*/
+#define FSTBLK_MAX_BUF_LEN                                                     \
+    FSTBLK_HEADER_LEN + FSTBLK_PRE_RSPKL_LEN + BLOCK_RSPKL_LEN +               \
+        FSTBLK_RSPERR_LEN + FSTBLK_RSPOK_LEN + (BLOCK_ERR_LEN * MAXBLOCKOPS) + \
+        sizeof(int) /* snapinfo_outrc */ + ERRSTAT_LEN /* iq->errstat */ +     \
+        sizeof(int) /* ? */ + sizeof(int) /* ? */ +                            \
```

Comment:
One of the integers is the epoch time of the blkseq. Not sure about the other one.

Reply:
thanks!